source/js/components/pulse-project-list/pulse-project-list.jsx

```diff
@@ -63,10 +63,24 @@ class PulseProjectList extends Component {
     projectXHR.send();
   }
 
+  // Giving users ability to link to pulse objects using an anchor link
+  // and having it render in the right place after recieving data.
+  scrollToLinkedPulseObject(){
+    const idOfLinkedPulseObject = (window.location.hash).substr(1)
+    const linkedPulseObject = document.getElementById(idOfLinkedPulseObject);
+    linkedPulseObject.scrollIntoView()
```

Comment:
and then let's make sure to only scroll into view if there _was_ an element =)

```suggestion
    linkedPulseObject?.scrollIntoView()
```